site/source/docs/building_from_source/building_emscripten_from_source_using_the_sdk.rst

```diff
@@ -38,8 +38,6 @@ The :ref:`emsdk` controls which tool and SDK "targets" are installed and activat
 	# Set the "incoming SDK" as the active version.
 	./emsdk activate sdk-incoming-64bit
 
-.. todo:: **HamishW** Check whether the bug (https://github.com/juj/emsdk/issues/13) is fixed and remove the windows specific line if it is.
-
```

Comment:
this was solved a while back, and the windows-specific line it refers to does not seem to exit (also, this was just a comment, never emitted to the site)